examples/rust-wasm-cross/.test.sh

```diff
@@ -4,7 +4,8 @@ cargo --version
 rustc --version
 
 if [[ "$(uname)" == "Darwin" ]]; then
-  echo "$RUSTFLAGS" | grep -- "-L framework=$DEVENV_PROFILE/Library/Frameworks"
+  # See env.RUSTFLAGS in devenv.nix
+  # echo "$RUSTFLAGS" | grep -- "-L framework=$DEVENV_PROFILE/Library/Frameworks"
```

Comment:
we should still assert this when not doing wasm

Reply:
This example is wasm-only though

Reply:
Ah! Would it make sense to just remove this?

Reply:
I guess this got copy-pasted over from the regular rust test. 🤷